app/src/main/java/io/gnosis/safe/notifications/models/PushNotification.kt

```diff
@@ -12,30 +12,34 @@ sealed class PushNotification(
 
     data class ConfirmationRequest(
         override val safe: Solidity.Address,
-        val safeTxHash: String
+        val safeTxHash: String,
+        val chainId: Int
```

Comment:
chainId should be in base class

```kotlin
abstract val chainId: Int
abstract val safe: Solidity.Address
```
In subclasses I would put chainId as a first parameter (first we identify chain, then safe on this chain, then transaction for this safe. This sequence of parameters seems sensible to me)